src/app/src/components/FacilityListItemsDetailedTableRowCell.jsx

```diff
@@ -71,7 +71,7 @@ export default function FacilityListItemsDetailedTableRowCell({
                 </div>
                 {
                     data.map((item, index) => (
-                        <Fragment key={hasActions ? item.id : item}>
+                        <Fragment key={item.id ? item.id : item}>
```

Comment:
This line was causing the main facility to receive an object as a key, instead of an id, causing a key warning in the console.